yb-voyager/src/tgtdb/event.go

```diff
@@ -69,12 +73,12 @@ func (event *Event) getUpdateStmt(targetSchema string) string {
 	}
 	var setClauses []string
 	for column, value := range event.Fields {
-		setClauses = append(setClauses, fmt.Sprintf("%s = %s", column, value))
+		setClauses = append(setClauses, fmt.Sprintf("%s = '%s'", column, value))
```

Comment:
Not all values should be quoted! The `ValueConverter.ConvertEvent()` must put quotes only if required by the value's type (e.g. string).